schema/objects/StockPlan.schema.json

```diff
@@ -41,9 +41,9 @@
     "stock_class_ids": {
       "description": "Identifiers of StockClass objects this plan is composed of",
       "type": "array",
+      "minItems": 1,
       "items": {
-        "type": "string",
-        "minItems": 1
```

Comment:
The validator was showing a warning about these two occurrences of minItems that were in the wrong spot
```
strict mode: missing type "array" for keyword "minItems" at "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/objects/StockPlan.schema.json#/properties/stock_class_ids/items" (strictTypes)